platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/implementing_custom_tasks.adoc

```diff
@@ -340,7 +340,7 @@ It's also beneficial to make our task class `abstract` because Gradle will handl
 ----
 import org.gradle.api.DefaultTask
 
-public abstract class GenerateReportTask : DefaultTask() {
+abstract class GenerateReportTask : DefaultTask() {
```

Comment:
this has no effect, public is default visibility in Kotlin.

Reply:
I removed `public` so it should be ok. Or you had something else in your mind?

Reply:
No, was just confused about the other change I thought was Java not Groovy